openbas-api/src/main/java/io/openbas/rest/channel/output/ArticleOutput.java

```diff
@@ -0,0 +1,86 @@
+package io.openbas.rest.channel.output;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.openbas.database.model.Article;
+import io.openbas.database.model.Document;
+import io.openbas.database.model.Exercise;
+import io.openbas.database.model.Scenario;
+import jakarta.persistence.Transient;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Optional.ofNullable;
+
+@Data
+public class ArticleOutput {
+
+  @JsonProperty("article_id")
+  @NotBlank
+  private String id;
+
+  @JsonProperty("article_name")
+  private String name;
+
+  @JsonProperty("article_content")
+  private String content;
+
+  @JsonProperty("article_author")
+  private String author;
+
+  @JsonProperty("article_shares")
+  private Integer shares;
+
+  @JsonProperty("article_likes")
+  private Integer likes;
+
+  @JsonProperty("article_comments")
+  private Integer comments;
+
+  @JsonProperty("article_exercise")
+  private String exercise;
+
+  @JsonProperty("article_scenario")
+  private String scenario;
+
+  @JsonProperty("article_channel")
+  @NotBlank
+  private String channel;
+
+  @JsonProperty("article_documents")
+  private List<String> documents = new ArrayList<>();
+
+  @Transient
+  private Instant virtualPublication;
+
+  @JsonProperty("article_virtual_publication")
+  public Instant getVirtualPublication() {
+    return this.virtualPublication;
+  }
+
+  @JsonProperty("article_is_scheduled")
+  public boolean isScheduledPublication() {
+    return this.virtualPublication != null;
+  }
+
+  public static ArticleOutput from(@org.jetbrains.annotations.NotNull final Article article) {
```

Comment:
I think you should use jakarta.validation.constraints.NotNull instead of org.jetbrains.annotations.NotNull